meta-balena-common/recipes-devtools/go/go-target.inc

```diff
@@ -1,4 +1,4 @@
-DEPENDS = "virtual/${TARGET_PREFIX}go go-native"
+DEPENDS = "virtual/${TUNE_PKGARCH}-go go-native"
```

Comment:
See comments above, `TARGET_PREFIX` seems the correct variable to use here.

Reply:
I am not sure about any of this, I just pulled in the changes from poky here... See https://git.yoctoproject.org/cgit/cgit.cgi/poky/log/meta/classes/go.bbclass?h=zeus

Reply:
You're right, after digging a bit this is explained in https://patchwork.openembedded.org/patch/156028/